android/app-project-newm/src/main/java/io/projectnewm/screens/Screen.kt

```diff
@@ -7,6 +7,7 @@ sealed class Screen(val route: String) {
     object TribeRoot : Screen("tribe-root")
     object StarsRoot : Screen("stars-root")
     object WalletRoot : Screen("wallet-root")
+    object NowPlayingRoot : Screen("now-playing-root")
```

Comment:
Do we need the NowPlaying as a root as well as the NowPlayingLanding?

Reply:
Honestly not sure I was just following the pattern there :D I can play around with it in a follow up. I was wondering the same